packages/react-scripts/README.md

```diff
@@ -1,7 +1,92 @@
-# react-scripts
+# backpack-react-scripts
 
-This package includes scripts and configuration used by [Create React App](https://github.com/facebook/create-react-app).<br>
-Please refer to its documentation:
+### **Important:** The currently supported version of **CRA** by `backpack-react-scripts` is up to `v3.4.1`. Versions above this will not work.
 
-- [Getting Started](https://facebook.github.io/create-react-app/docs/getting-started) – How to create a new app.
-- [User Guide](https://facebook.github.io/create-react-app/) – How to develop apps bootstrapped with Create React App.
+This package is a fork of [Create React App](https://github.com/facebookincubator/create-react-app) (specifically the
+`react-scripts` package). It's intended to be used in conjuction with `create-react-app` like so:
+
+```sh
+npx create-react-app@3.4.1 my-app --scripts-version=backpack-react-scripts --template @skyscanner/backpack --use-npm
```

Comment:
is this command still accurate? do we need the @3.4.1 annotation?

Reply:
See comment above